### ISSUE TITLE
Update wiki

### ISSUE THREAD

**Issue by michaeljs1990** [ispc/ispc#1192]
Is it still recommended to use llvm 3.3 it seems like the entire wiki is a few years out of date.

**Comment by dbabokin** [comment]
Yes, wiki is a little bit out of date. Recommended version of LLVM is 3.8.

**Comment by RoyiAvital** [comment]
Are there plans to update the Wiki?

It would be great to have Users Samples section there.

**Comment by dbabokin** [comment]
Yes, we still plan to update the wiki, it's just not the top priority task at the moment.

I'm not sure what User Samples section you'd like to see on the wiki. I think examples from the package do pretty good job, plus examples from the documentation. If you feel that we have not enough examples covering your problem domain, you are welcome to contribute examples, which you find relevant.

**Comment by aneshlya** [comment]
@nurmukhametov , I think it can be closed as completed.
